chart/templates/configmap-etcd-config.yaml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
<del>Rename it to `etcd-config-configmap.yaml`.</del> Actually merge this with `etcd-bootstrap-configmap.yaml`.